default.nix

```diff
@@ -2,7 +2,11 @@
 , inputs ? import ./flake.lock.nix
 , nixpkgs ? import inputs.nixpkgs {
     inherit system;
-    config = { allowUnfree = true; };
+    config = {
+      allowUnfree = true;
+      cudaSupport = true;
```

Comment:
How should we handle `cudaArchList`? Many packages have their own defaults, they sometimes use different names (e.g. `cudaArchList = [ "8.6+PTX" ];` in pytorch is `cudaCapabilities = [ "compute_86" ]` in jaxlib), and choose different ranges of compute versions (e.g. pytorch defaults include 8.6, whereas jaxlib by default includes code for up to 80), which means the end user will likely have to overlay some derivations and cause a cache-miss

Reply:
Could it actually be better, cache-wise, to build everything for a single target compute, that the end-user would set through something like `config.cudaArchList`? We'd run the CI over a range of compute versions then

Reply:
Ok I opened https://github.com/NixOS/nixpkgs/issues/163700.